themes/parrot/parrot.theme.bash

```diff
@@ -0,0 +1,38 @@
+#
+# ~/.bashrc
+#
+
+# git branch parser
+function parse_git_branch() {
+    echo -e "\033[1;34m$(git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/(\1)/')\033[0m"
+}
+
+function parse_git_branch_no_color() {
+    echo -e "$(git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/(\1)/')"
+}
+
+function promp() {
```

Comment:
Where is this function used?

Reply:
It will create a prompt like Parrot OS Terminal

![](https://community-cdn.static.fzbx.io/uploads/default/original/2X/6/6aae4c74eaa540cce4e98b0527f6375df8a86423.png)

Reply:
and the function promp is used to set PS1 to new prompt theme